Tasks/Maslov/task2/Dz2.py

```diff
@@ -0,0 +1,43 @@
+
+import datetime
+
+a = datetime.datetime.now()
+print(a)
+
+h = {0: "ноль", 1: "один", 2: "два", 3: "три", 4: "четыре", 5: "пять", 6: "шесть",
+     7: "семь", 8: "восемь", 9: "девять", 10: "десять", 11: "одиннадцать", 12: "двенадцать", 13: "тринадцать",
+     14: "четырнадцать", 15: "пятнадцать", 16: "шестнадцать", 17: "семнадцать", 18: "восемнадцать",
+     19: "девятнадцать", 20: "двадцать", 21: "двадцать один", 22: "двадцать два", 23: "двадцать три",
+     24: "двадцать четыре"}
+m = {0: "ровно", 1: "один", 2: "два", 3: "три", 4: "четыре", 5: "пять", 6: "шесть",
```

Comment:
![image](https://user-images.githubusercontent.com/57627287/151111505-4345fa4f-f439-4ed6-9974-f2facd0c81fa.png)
I think it's not exactly correct